src/meetingsession/MeetingSessionConfiguration.ts

```diff
@@ -172,4 +180,34 @@ export default class MeetingSessionConfiguration {
       this.credentials ? this.credentials.attendeeId : null
     );
   }
+
+  private toLowerCasePropertyNames(jsonObject: any) { // eslint-disable-line
```

Comment:
Two conversions-from an object to Map to JSON-are not required. Use this function that also handles a non-object and an array type (e.g. an object in an array).

```
// eslint-disable-next-line @typescript-eslint/no-explicit-any
private toLowerCasePropertyNames(input: any): any {
  if (typeof input !== 'object') {
    return input;
  } else if (Array.isArray(input)) {
    return input.map(toLowerCasePropertyNames);
  };
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  return Object.keys(input).reduce((result: any, key: string) => {
    const value = input[key];
    const newValue = typeof value === 'object' ? toLowerCasePropertyNames(value) : value;
    result[key.toLowerCase()] = newValue;
    return result;
  }, {});
};
```

Reply:
1. [Click here to see the TypeScript playground](https://www.typescriptlang.org/play?#code/PTAEFMGcBsEsDsAuBaAJrSBDARtcz5wAPFOQ0AAUQE8AHKAYwCdZaUozFh4B7ZY2nAawUmeNQBQAMwCu8Bolg94oRDwAyPAO7gmAYUyRwABSY96TGgDlMAWygAKBLRmIAXKDHUAlB6+gAbwlQENBYKVAHGnoeCOdXUABCAF5k0AByHmwAK3AFdO9A4NCSpnBEGSYVeMQAbmKQgF8IaCMwiIcAQSYmTGoAOgxu3uoneBdEb0KgktLyyurx137bTFoojW1dAyNTc11rOyhvepLG09CQCBgEFHQsXHxCEmQycEpoxhY2fhukbj4AiEImQXgaoDKFSqoAA8jk8oh+gBrcDUSBjCbefplVAyBjgBwOMqQGTQdyecQAGlAKOoHkgiBY8AA5oVkgA+IqzUIMZQM0AAN0w0Bk7zSNQA2rSALoXbm8+D8whaABqwtFoDSn1igvVYtSGSyuXyoAA-KpNjp9IYTGYLId7OihSLwIUPM7RXLZsTSYgpaj+mpNFadgTvNLNaBlWqXV65lCVD6yV7GtSAo0ThJzhIJAr+TVIzNQAAiABSPAQAEl4FIeMWPEWQsWALLgcoIZn1rmzFttxQsruNnut9ssyuoLvFgAcU4AzAB2BizgBsAAZkABOVDYKfIAAsAFZ5wBGTeYPez-izg8HqSoKcAJlQmFQy+LlPBJWLAFESLp4MKI79sy46TgAyogYioM+0BSEi2QTh+3JNq26CYMY0CYPi9hIIOn49p0MjoDwAASPAMgAqkw0CTg+x7gMeqDgFI85SMee5bqu4BPpgmDHses6zqgB7XqgDBSI+yIrA+-QyFox79GstD9AwAAWsD2IpWiQG4s57vOU7vvhX6EcRABiwrQNgWFIlRNEeMW2k6SAqmYEQTDKbYMlyQpSkqepmmYNpbh7hewAMJZkDADOC5Lmum7bruh4nmeF5Xjed6Ps+r5GchoTFmBzBtvAAAimCQXZk5OW4IDYCItBaNJsnyYptDKWpGngFpOmhbOwACg+wCQEV4DwNFc6Liu65bju+5HqeG7npe4DXre948TlSF5SWhVlKNYGuUyzKVQ51W1fVjVec1vltf5nXdSFYUDUNI1jTFk3xTNSXzaly2rZlG1vlteUFa9KqwOAWgdidJZncAdWIA1TU+a17UBV1QU9WF2nhcohAKKatCGMNPBMck8CktAABkAoQ1aAD6MgyLAqDk5TVMABrIAAQiIJEyNgyAGNA0DIOOyTvXF02JXNKWLWlK0Zet2VA8Z+VgbAzIAWQx3UVVkDOUNmva8jLV+R1gU4wqjI8NA42xVNCWzclC1Lela1ZS+qvbSWAAqCx6MoNvQDDxaqYgiOGwwDDtcpoo3Wj92Y-1g0JvTRgG0oirFmrqZq72aEAErgMyWeTjIkDIDoDLIMeOfco04J5yZEejUx4B4dyxadK38Dt53yE-n+VTChRRhMKBDl7tgx44POO4AMTPlI0BIoYqDZLlg894gbdtpPJasQwG7T1O66rs+l6zuAHFoMux5SLLmAPlO4B8YtmBb135YIL7PAovAScVgHwAC1sgwl9oXWAMISo92bAAcULqpZs1BVwAC8QG+wAIoHnUAAdSwQ+ZseCACaiArAlQorOEhvtKx7hAXg38MI8FmVUjCbI34iG+2ZA+KwvsKKrnUL7EBsASHZCweQ+BKpsggLAseVSNDWFCI3ggiiRAYTwIoloTBlZZzNiwfXWYjczjFEbtmXMfJbZdWgDwZkGxgzbBtHse01AbCOgxK4KYtQgA)
2. Open the browser console and click on "Run." (Or view the Logs tab in the right panel.)